server/controllers/finance/reports/invoicedReceivedStock/index.js

```diff
@@ -0,0 +1,181 @@
+const _ = require('lodash');
+const db = require('../../../../lib/db');
+const ReportManager = require('../../../../lib/ReportManager');
+const Patients = require('../../../medical/patients');
+
+const TEMPLATE = './server/controllers/finance/reports/invoicedReceivedStock/report.handlebars';
+// expose to the API
+exports.report = report;
+
+// default report parameters
+const DEFAULT_PARAMS = {
+  csvKey : 'invoiced_received_stock',
+  filename : 'REPORT.COMPARE_INVOICED_RECEIVED.TITLE',
+  orientation : 'landscape',
```

Comment:
I think this report makes more sense in a `portrait` than landscape.  Here is an example of my tests.

![image](https://user-images.githubusercontent.com/896472/81284797-80295100-9056-11ea-9af5-5202b675f7a1.png)

Reply:
Or maybe just removing the padding would address this, actually.

Reply:
> Or maybe just removing the padding would address this, actually.

I also just noticed that when you download a report, the shades of gray disappear, with you noticing this problem 
![image](https://user-images.githubusercontent.com/8035588/81390977-4454c100-9114-11ea-818f-c8bc536e2639.png)

Reply:
Yeah, that has to do with #4382.  It's why I was hoping for a solution that didn't involve `!important;` everywhere.  Don't worry about that for now.